api/views.go

```diff
@@ -82,3 +83,37 @@ func (c *Views) List() ([]ViewListItem, error) {
 
 	return q.View, graphqlErr
 }
+
+type ViewConnectionInput struct {
+	RepositoryName graphql.String
+	Filter graphql.String
+}
+
+func (c *Views) Create(name, description string, connections map[string]string) error {
+	var m struct {
+		CreateView struct {
+			Name string
+			Description string
+		} `graphql:"createView(name: $name, description: $description, connections: $connections)"`
```

Comment:
Can we somehow let the graphql package handle the serialization. There are some [examples](https://github.com/shurcooL/graphql) here that might help: